apps/src/templates/studioHomepages/TwoColumnActionBlock.jsx

```diff
@@ -3,10 +3,16 @@ import color from "../../util/color";
 import GridContainer from './GridContainer';
```

Comment:
Let's remember is sync when #19042 is ready so I don't break anything with the `TwoColumActionBlock`.